packages/terra-consumer-icon/scripts/src/migrate-svg/csvObject.js

```diff
@@ -0,0 +1,15 @@
+import { IconPathDetails } from '../config';
+
+class csvObject {
+  constructor(name, filepath, themeable, bidi) {
+    this.name = name;
+    // use current directory svg path
+    this.svgSrc = `tmp/svg/${filepath}`;
```

Comment:
this is the temporary svg source until we get final mapping and the released consumer-icons package.